### PR TITLE
feat(reference/kubernetes): recreate annotation docs

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: reference/providers/kubernetes-v2/index.md [hunk 1]
```diff
@@ -150,6 +150,19 @@ command](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands
   If instead Spinnaker is deploying ReplicaSets directly without a Deployment,
   this annotation does the job.
 
+* `strategy.spinnaker.io/recreate`
+
+  As of Spinnaker 1.13, you can force Spinnaker to delete a resource (if it
+  already exists) before creating it again. This is useful for kinds such
+  as [`Job`](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/)
```
Comment (dorbin) [inline reference/providers/kubernetes-v2/index.md]: Add comma  after the linked text `Job`.

So "...useful for kinds such as Job, which cannot be edited..."

File: reference/providers/kubernetes-v2/index.md [hunk 2]
```diff
@@ -150,6 +150,19 @@ command](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands
...
+  which cannot be edited once created, or must be recreated to run again.
+  
+  When set to `'true'` for a versioned resource, this will only recreate your
```
Comment (dorbin) [inline reference/providers/kubernetes-v2/index.md]: Here too.

File: reference/providers/kubernetes-v2/index.md [hunk 3]
```diff
@@ -150,6 +150,19 @@ command](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands
   If instead Spinnaker is deploying ReplicaSets directly without a Deployment,
   this annotation does the job.
 
+* `strategy.spinnaker.io/recreate`
+
+  As of Spinnaker 1.13, you can force Spinnaker to delete a resource (if it
+  already exists) before creating it again. This is useful for kinds such
+  as [`Job`](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/)
+  which cannot be edited once created, or must be recreated to run again.
```
Comment (dorbin) [inline reference/providers/kubernetes-v2/index.md]: Hyphenate "re-created." The hyphen distinguishes re-creation from the set of avocations we think of as "recreation"&mdash;rock climbing, brewery touring, competitive eating...

**Review by dorbin — COMMENTED** [review]
LGTM, after a couple of small changes

File: reference/providers/kubernetes-v2/index.md (hunk 3 above)
Comment (lwander) [reply reference/providers/kubernetes-v2/index.md]: doi! Thanks

**Review by dorbin — APPROVED** [review]

**Pull request merged**